backend/src/run.py

```diff
@@ -190,11 +193,45 @@ async def run_individual(request: Request):
         logger.info(f"Using device: {os.environ['device']}")
         # Create node based on given category/name information
         node_instance = NodeFactory.create_node(full_data["schemaId"])
+
+        # Enforce that all inputs match the expected input schema
+        enforced_inputs = []
```

Comment:
I don't like that all this executor code is now here. We should probably make a follow up PR adding model outputs. Then we could get rid of this, because we wouldn't need `get_extra_data` anymore.

Reply:
On that note: Is the `get_extra_data` in Load Image even used anymore?

Reply:
I've already finished the model outputs stuff locally. Just waiting for this to be finished before I make that PR so I can rebase. And no, that get extra data is no longer used. I'll remove it.